2021-09-07 20:03:23 WARNING (MainThread) [homeassistant.components.sensor] Entity sensor.cpu_temperature (<class 'homeassistant.components.command_line.sensor.CommandSensor'>) with device_class None reports a temperature in °C which will be converted to °F. Temperature conversion for entities without correct device_class is deprecated and will be removed from Home Assistant Core 2022.3. Please update your configuration if device_class is manually configured, otherwise create a bug report at https://github.com/home-assistant/core/issues?q=is%3Aopen+is%3Aissue+label%3A%22integration%3A+command_line%22
2021-09-07 20:03:35 WARNING (MainThread) [homeassistant.config_entries] Config entry '49" TCL Roku TV' for roku integration not ready yet: Invalid response from API: Error occurred while communicating with device; Retrying in background
2021-09-07 20:03:35 WARNING (MainThread) [homeassistant.config_entries] Config entry 'TCL•Roku TV - YN00S3510999' for roku integration not ready yet: Invalid response from API: Error occurred while communicating with device; Retrying in background
2021-09-07 20:03:35 WARNING (MainThread) [homeassistant.config_entries] Config entry 'TCL•Roku TV - X000001GHXPY' for roku integration not ready yet: Invalid response from API: Error occurred while communicating with device; Retrying in background
2021-09-07 20:03:38 WARNING (MainThread) [homeassistant.components.esphome] Can't connect to ESPHome API for ir_code_capture (192.168.42.146): Error connecting to ('192.168.42.146', 6053): [Errno 113] Connect call failed ('192.168.42.146', 6053)
2021-09-07 20:03:38 WARNING (MainThread) [homeassistant.components.esphome] Can't connect to ESPHome API for batt_volt_test (192.168.42.125): Error connecting to ('192.168.42.125', 6053): [Errno 113] Connect call failed ('192.168.42.125', 6053)
2021-09-07 20:03:38 WARNING (MainThread) [homeassistant.components.esphome] Can't connect to ESPHome API for four_channel_relay (192.168.42.132): Error connecting to ('192.168.42.132', 6053): [Errno 113] Connect call failed ('192.168.42.132', 6053)
2021-09-07 20:03:38 WARNING (MainThread) [homeassistant.components.esphome] Can't connect to ESPHome API for moisture_sensor_prototype (192.168.42.133): Error connecting to ('192.168.42.133', 6053): [Errno 113] Connect call failed ('192.168.42.133', 6053)
2021-09-07 20:03:38 WARNING (MainThread) [homeassistant.components.esphome] Can't connect to ESPHome API for beer_fridge (192.168.42.142): Error connecting to ('192.168.42.142', 6053): [Errno 113] Connect call failed ('192.168.42.142', 6053)
2021-09-08 06:57:49 ERROR (MainThread) [homeassistant.components.hassio.handler] /ingress/validate_session return code 401
2021-09-08 06:57:49 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /ingress/validate_session - 
2021-09-08 06:57:49 ERROR (MainThread) [homeassistant.components.hassio.handler] /ingress/validate_session return code 401
2021-09-08 06:57:49 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /ingress/validate_session - 
2021-09-08 06:57:49 ERROR (MainThread) [homeassistant.components.hassio.handler] /ingress/validate_session return code 401
2021-09-08 06:57:49 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /ingress/validate_session - 
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio.handler] /ingress/validate_session return code 401
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /ingress/validate_session - 
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio.handler] /ingress/validate_session return code 401
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /ingress/validate_session - 
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio.handler] /ingress/validate_session return code 401
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /ingress/validate_session - 
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio.handler] /ingress/validate_session return code 401
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /ingress/validate_session - 
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio.handler] /ingress/validate_session return code 401
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /ingress/validate_session - 
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio.handler] /ingress/validate_session return code 401
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /ingress/validate_session - 
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio.handler] /ingress/validate_session return code 401
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /ingress/validate_session - 
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio.handler] /ingress/validate_session return code 401
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /ingress/validate_session - 
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio.handler] /ingress/validate_session return code 401
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /ingress/validate_session - 
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio.handler] /ingress/validate_session return code 401
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /ingress/validate_session - 
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio.handler] /ingress/validate_session return code 401
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /ingress/validate_session - 
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio.handler] /ingress/validate_session return code 401
2021-09-08 06:57:50 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /ingress/validate_session - 
2021-09-08 06:59:47 WARNING (MainThread) [homeassistant.components.esphome] Can't connect to ESPHome API for ir_code_capture (192.168.42.146): Disconnected while waiting for API response!
2021-09-08 09:21:54 WARNING (MainThread) [homeassistant.components.esphome] Can't connect to ESPHome API for ir_code_capture (192.168.42.146): Timeout while connecting to ('192.168.42.146', 6053)
2021-09-08 22:07:42 WARNING (MainThread) [homeassistant.components.esphome] Can't connect to ESPHome API for water_softener (192.168.42.135): Timeout while connecting to ('192.168.42.135', 6053)
2021-09-08 22:07:43 ERROR (MainThread) [homeassistant.components.brother] Error fetching brother data: No SNMP response received before timeout
2021-09-08 22:07:54 WARNING (MainThread) [homeassistant.components.esphome] Can't connect to ESPHome API for wifi_bulb_1 (192.168.42.134): Timeout while connecting to ('192.168.42.134', 6053)
2021-09-09 13:19:11 WARNING (MainThread) [homeassistant.components.esphome] Can't connect to ESPHome API for ir_code_capture (192.168.42.146): Error connecting to ('192.168.42.146', 6053): [Errno 113] Connect call failed ('192.168.42.146', 6053)
2021-09-09 13:23:19 ERROR (MainThread) [homeassistant.components.hassio.handler] /ingress/validate_session return code 401
2021-09-09 13:23:19 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /ingress/validate_session - 
2021-09-09 13:23:19 ERROR (MainThread) [homeassistant.components.hassio.handler] /ingress/validate_session return code 401
2021-09-09 13:23:19 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /ingress/validate_session - 
2021-09-09 13:23:35 ERROR (MainThread) [homeassistant.components.hassio.handler] /ingress/validate_session return code 401
2021-09-09 13:23:35 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /ingress/validate_session - 
2021-09-09 13:24:28 WARNING (MainThread) [homeassistant.components.esphome] Can't connect to ESPHome API for ir_code_capture (192.168.42.146): Disconnected while waiting for API response!
2021-09-09 13:37:32 WARNING (MainThread) [homeassistant.components.esphome] Can't connect to ESPHome API for ir_code_capture (192.168.42.146): Timeout while connecting to ('192.168.42.146', 6053)
2021-09-09 13:43:31 WARNING (MainThread) [homeassistant.components.esphome] Can't connect to ESPHome API for ir_code_capture (192.168.42.146): Error connecting to ('192.168.42.146', 6053): [Errno 113] Connect call failed ('192.168.42.146', 6053)
2021-09-09 22:10:19 ERROR (MainThread) [homeassistant.components.roku.coordinator] Error fetching roku data: Invalid response from API: Timeout occurred while connecting to device
2021-09-10 08:20:59 ERROR (MainThread) [frontend.js.latest.202108300] http://192.168.42.115:8123/frontend_latest/core.9dd583b2.js:1:7650 Uncaught Error: Suspend promise not set
2021-09-10 21:39:13 WARNING (MainThread) [homeassistant.components.esphome] Can't connect to ESPHome API for ir_code_capture (192.168.42.146): Timeout while connecting to ('192.168.42.146', 6053)
2021-09-10 21:41:26 WARNING (MainThread) [homeassistant.components.esphome] Can't connect to ESPHome API for ir_code_capture (192.168.42.146): Timeout while connecting to ('192.168.42.146', 6053)
2021-09-10 21:45:19 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/hassio/ingress.py", line 250, in _websocket_forward
    await ws_to.send_bytes(msg.data)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_ws.py", line 307, in send_bytes
    await self._writer.send(data, binary=True, compress=compress)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/http_websocket.py", line 685, in send
    await self._send_frame(message, WSMsgType.BINARY, compress)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/http_websocket.py", line 650, in _send_frame
    self._write(header + message)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/http_websocket.py", line 660, in _write
    raise ConnectionResetError("Cannot write to closing transport")
ConnectionResetError: Cannot write to closing transport
2021-09-10 21:45:19 ERROR (MainThread) [homeassistant.components.hassio.handler] /ingress/validate_session return code 401
2021-09-10 21:45:19 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /ingress/validate_session - 
2021-09-10 21:50:55 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/hassio/ingress.py", line 250, in _websocket_forward
    await ws_to.send_bytes(msg.data)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_ws.py", line 307, in send_bytes
    await self._writer.send(data, binary=True, compress=compress)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/http_websocket.py", line 685, in send
    await self._send_frame(message, WSMsgType.BINARY, compress)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/http_websocket.py", line 650, in _send_frame
    self._write(header + message)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/http_websocket.py", line 660, in _write
    raise ConnectionResetError("Cannot write to closing transport")
ConnectionResetError: Cannot write to closing transport
2021-09-10 22:22:35 WARNING (MainThread) [homeassistant.components.esphome] Can't connect to ESPHome API for ir_code_capture (192.168.42.146): Timeout while connecting to ('192.168.42.146', 6053)
2021-09-11 18:08:43 WARNING (MainThread) [homeassistant.components.esphome] Can't connect to ESPHome API for ir_code_capture (192.168.42.146): Disconnected while waiting for API response!
2021-09-11 18:36:06 ERROR (MainThread) [homeassistant.components.hassio.handler] Client error on /core/update request Server disconnected
2021-09-11 18:36:06 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /core/update - 
